.circleci/config.yml

```diff
@@ -68,9 +68,6 @@ browsers_unit_tests: &browsers_unit_tests
     - run:
         name: Install Dependencies
         command: yarn install
-    - run:
-        name: Compile code
-        command: yarn compile
```

Comment:
Could this also be removed in `node_unit_tests`?

Reply:
I tried, but build is failing after that https://app.circleci.com/jobs/github/open-telemetry/opentelemetry-js/7777

Reply:
I believe the issue is a permissions one. https://docs.npmjs.com/misc/scripts#user

Try enabling the `unsafe-perm` flag and removing yarn compile from the `node_unit_tests`